examples/prepare_gp_optimizer.py

```diff
@@ -42,7 +42,7 @@
 _ = make_dir_tree(datetime.datetime.now().year, base_path=root_dir)
 
 
-connection = SirepoBluesky("http://localhost:8000")
+connection = SirepoBluesky("http://localhost:8001")
```

Comment:
Maybe instead we'd like to configure tiled to use a non-standard port, following the instructions at https://blueskyproject.io/tiled/how-to/profiles.html#create-and-use-a-profile. I think we can use `localhost:8001` for tiled rather than for Sirepo as it is already used in CI, etc.

Reply:
Yeah we should do this at some point